plugins/inputs/nstat/README.md

```diff
@@ -0,0 +1,3 @@
+## Nstat input plugin 
```

Comment:
readme will need to be improved, 
model the readme after https://github.com/influxdata/telegraf/blob/master/plugins/inputs/EXAMPLE_README.md